CMakeLists.txt

```
@@ -207,6 +212,41 @@ target_compile_definitions(remill_settings INTERFACE
   "REMILL_BUILD_SEMANTICS_DIR_PPC64_32ADDR=\"${REMILL_BUILD_SEMANTICS_DIR_PPC64_32ADDR}\""
 )
 
+set(ghidra_patch_user "github-actions[bot]")
+set(ghidra_patch_email "41898282+github-actions[bot]@users.noreply.github.com")
+
+# pinned stable patches list
+set(ghidra_patches
+  PATCH_COMMAND "${GIT_EXECUTABLE}" config user.name "${ghidra_patch_user}" &&
+  "${GIT_EXECUTABLE}" config user.email "${ghidra_patch_email}" &&
+  "${GIT_EXECUTABLE}" am --ignore-space-change --ignore-whitespace --no-gpg-sign)
+list(APPEND ghidra_patches ${sleigh_ADDITIONAL_PATCHES})
+
+FetchContent_Declare(ghidra-fork
+  GIT_REPOSITORY https://github.com/trail-of-forks/ghidra.git
+  GIT_TAG a5214a7
+  ${ghidra_patches}
+)
+
+FetchContent_MakeAvailable(ghidra-fork)
+
+if(SLEIGH_EXECUTABLE)
+  set(sleigh_compiler "${SLEIGH_EXECUTABLE}")
+else()
+  set(sleigh_compiler "$<TARGET_FILE:sleigh::sleigh>")
+endif()
+
+sleigh_compile(
+  TARGET ppc_e200_spec
+  COMPILER "${sleigh_compiler}"
+  SLASPEC "${ghidra-fork_SOURCE_DIR}/Ghidra/Processors/PowerPC/data/languages/ppc_32_e200_be.slaspec"
+  LOG_FILE "${CMAKE_BINARY_DIR}/specfiles/ppc_32_e200_be.sla.log"
+  OUT_FILE "${CMAKE_BINARY_DIR}/specfiles/ppc_32_e200_be.sla"
```

```suggestion
  LOG_FILE "${sleigh_BINARY_DIR}/sleighspecs/spec_build_logs/ppc_32_e200_be.sla.log"
  OUT_FILE "${sleigh_BINARY_DIR}/specfiles/Ghidra/Processors/PowerPC/data/languages/ppc_32_e200_be.sla"
```
this will bulid the sleigh spec to the same directory as the other spec files so they can be found without installation